### PR TITLE
Register y.is-a.dev

### PR DESCRIPTION
**Pull request by jalamangkara**
Added `y.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please use a valid record.

File: domains/api.grab.com.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "jalamangkara",
+        "email": "jalamangkara700@gmail.com"
+    },
+
+    "record": {
+        "A": ["y"]
```
Comment (WilliamDavidHarrison) [inline domains/api.grab.com.json:8]: Please use a valid record.

File: domains/y.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "jalamangkara",
+        "email": "jalamangkara700@gmail.com"
+    },
+
+    "record": {
+        "A": ["10"]
```
Comment (WilliamDavidHarrison) [inline domains/y.json:8]: Please use a valid record.

File: domains/api.grab.com.json
Comment (WilliamDavidHarrison) [inline domains/api.grab.com.json:1]: You will need to register `com.is-a.dev` & `grab.com.is-a.dev` to be able to register `api.grab.com.is-a.dev`

File: domains/y.json
Comment (WilliamDavidHarrison) [inline domains/y.json:1]: Your subdomain name must be 2 characters or more.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
I am closing this PR due to no response and more invalid domains.

**Pull request closed without merging**